depends/funcs.mk

```diff
@@ -37,7 +37,7 @@ define vendor_crate_deps
   ( mkdir -p $$($(1)_download_dir)/$(1) && echo Vendoring dependencies for $(1)... && \
     tar -xf $(native_rust_cached) -C $$($(1)_download_dir) && \
     tar --strip-components=1 -xf $$($(1)_source_dir)/$(2) -C $$($(1)_download_dir)/$(1) && \
-	patch -p1 -d $$($(1)_download_dir)/$(1) < $(PATCHES_PATH)/$(1)/$(3) && \
+	cp $$($(1)_download_dir)/$(1)/$(3) $$($(1)_download_dir)/$(1)/Cargo.lock && \
```

Comment:
It's really difficult for me to understand whether this change is correct.

Reply:
That's automake for ya.

In order to build `cxxbridge` in Gitian, we need to vendor its dependencies duting the `fetch` stage. Previously there was no provided lockfile (that I was aware of), so we were using a patch file to apply one. However, the `cxx` repo (now) provides a lockfile in a subdir, so this is replacing the patch command with a "copy to root" command (with the exact path left to the caller). It's a little specialised to `cxxbridge` but that's fine because this whole function was written specifically for `cxxbridge`.